index.html

```diff
@@ -18,25 +18,57 @@ <h1 class="display-4 text-center">
         <i class="fas fa-book-open text-primary"></i> My<span class="text-primary">Book</span>List
       </h1>
       <form id="book-form">
-        <div class="form-group">
+        <div class="form-group mb-2">
           <label for="title">Title</label>
           <input type="text" id="title" class="form-control">
+          <span class="text small text-danger titleError">Title is required</span>
```

Comment:
Consider implementing client-side form validation to ensure that the title field is not left blank before submission.